pygeo/parameterization/DVGeo.py

```diff
@@ -1793,7 +1825,7 @@ def pointSetUpToDate(self, ptSetName):
         else:
             return True
 
-    def convertSensitivityToDict(self, dIdx, out1D=False):
+    def convertSensitivityToDict(self, dIdx, out1D=False, compName=False):
         """
```

Comment:
Please update docstring

Reply:
can you rename compName variable to something else? I suggest `useCompositeNames`. `comp` might be confused for "component", plus adding "use" makes it more clear that we expect a boolean instead of a list or a string